guides/user/pipeline/expressions/index.md

```diff
@@ -10,492 +10,356 @@ redirect_from:
+This example outputs the status of your Deploy stage:
 
-![](14.png)
-
-The deploy stage should look like this:
-
-![](15.png)
-
-
-When the pipeline runs, it will replace the expression with the name of the branch, creating a new cluster:
-
-![](16.png)
-
-### Conditionally execute pipeline behavior based on manual input
-
-For this use case, we're going to set up a pipeline with 3 conditions, and gate these conditions with a manual judgment stage.
-
-1. First, add a manual judgment stage and add three options.
-
-![](17.png)
-
-2. Now that I have these inputs set up, I create a new precondition stage. I add an expression to check that the input was 'clean up'
-
-![](18.png)
-
-My precondition stage should look like this.
-
-![](19.png)
-
-> **Fail pipeline**
->
-> For these type of workflows, you want to set *Fail Pipeline* as false when you're setting up your expression.
-
-3. Add stages that you want to execute based on this precondition. 
-
-4.  I do the same for the 'deploy another' input.
-
-5. My pipeline now looks like this:
-
-![](20.png)
-
-6. Go to the executions screen and run my pipeline. I should see the options show up in the Manual Judgment stage:
-
-![](21.png)
-
-If I select 'do nothing', which matches no preconditions, my pipeline should just stop:
+```
+{"result": "SUCCEEDED"}
+```
 
-![](22.png)
+If Spinnaker cannot evaluate the expression, the result includes error details.
+For example, if you forget the closing bracket in the example above, it outputs
+the following:
 
-If I choose 'deploy another', I can see that the stages gated by deploy another are executed. 
+```
+{  
+  "detail":
+    {  
+      "{ #stage(\"Deploy\").status.toString() ":
+        [  
+          {  
+            "description":
+              "Failed to evaluate [expression] Expression
+              [{ #stage( #root.execution, \"Deploy\").status.toString() ]
+              @0: No ending suffix '}' for expression starting at character 0:
+              { #stage( #root.execution, \"Deploy\").status.toString() ",
+            "exceptionType":"org.springframework.expression.ParseException",
+            "level":"ERROR",
+            "timestamp":1531254890849
+         }
+        ]
+   },
+  "result":"${#stage(\"Deploy\").status.toString() "
+}
+```
 
-![](23.png)
+Note that [Gate](https://github.com/spinnaker/gate) is the microservice that
+provides access to the Spinnaker API. If you're accessing Spinnaker via the
+default `localhost:9000`, make sure you're sending your API requests to Gate
+(defaults to `localhost:8084`).
 
-Similarly, you can see that when I chose 'cleaning up', that pipeline branch is the one that runs. 
+Note also that pipeline expressions may not behave exactly the same against the
+testing endpoint and in your pipeline. In your pipeline, each expression is
+anchored to the particular stage in which it was declared. However, when using
```

Comment:
Is "anchored" the right word here? 

"Each expression runs against the particular stage..." maybe?

Reply:
I'm not sure. Do you think "runs against" conveys that the scope of what is accessible in the expression is dependent on the stage? e.g., you can't access values of future stages, and there are specific ways to access the value of the current stage. I'm definitely not tied to "anchored," but communicating that is why I picked it.

Reply:
"is limited in scope to..." ?